apollo-router/src/plugin/test/mod.rs

```diff
@@ -153,6 +155,7 @@ impl PluginTestHarness {
         );
         let plugins = Arc::new(plugins);
 
+        let apq = APQLayer::with_cache(DeduplicatingCache::new(512).await);
```

Comment:
NIT: 512 should be a well named const or static.

Reply:
counter NIT: we should write down the motivation for that value somewhere :smile: 
But yeah, I agree

Reply:
fixed in 3813a24d